hatchery/cloudwatch.go

```diff
@@ -20,7 +20,10 @@ func (sess *CREDS) CreateLogGroup(LogGroupName string, creds *credentials.Creden
 
 	logGroup, err := c.DescribeLogGroups(describeLogGroupIn)
 	if err != nil {
-		Config.Logger.Printf("Error in DescribeLogGroup: %s", err)
+		Config.Logger.Errorw("Error in DescribeLogGroup",
+			"error", err,
+			"logGroupNamePrefix", LogGroupName,
```

Comment:
same comment about `logGroupNamePrefix` and `newLogGroup` in this file, and other custom fields in other files